frontend/src/modules/Shared/Shares/ShareEditForm.js

```diff
@@ -398,6 +399,14 @@ export const ShareEditForm = (props) => {
         {getExplanation(shareStatus)}
       </Typography>
       <Box>
+        <Box sx={{ p: 1 }}>
+          {sharedItems.nodes.find((item) => item.itemType === 'S3Bucket') && (
+            <Alert severity="warning" gutterBottom sx={{ mr: 1 }}>
+              Sharing S3Bucket gives Requestor read access to{' '}
+              <b>the entire S3 Bucket</b> superseding Table + Folder shares
```

Comment:
For Folder shares yes, but for table shares not 100%; yes you can access the bucket but not the glue table

Reply:
I updated the comment to be more verbose

While you cannot access the glue table you would have a workaround to access and download the underlying file in S3 when it comes to table shares